src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1306,14 +1298,12 @@ internal virtual HttpResponseMessage GetResponse(HttpClient client, HttpRequestM
                     Method = WebRequestMethod.Get;
                 }
 
-                // recreate the HttpClient with redirection enabled since the first call suppressed redirection
                 currentUri = new Uri(request.RequestUri, response.Headers.Location);
-                using (client = GetHttpClient(false))
-                using (HttpRequestMessage redirectRequest = GetRequest(currentUri, stripAuthorization:true))
+                // Continue to handle redirection
+                using (client = GetHttpClient(true))
```

Comment:
Usually we use named parameter `GetHttpClient(handleRedirect:true)` - it is more readable.